INSTALL.md

```diff
@@ -16,6 +16,6 @@ Download using the [GitHub .zip download](https://github.com/getomni/alacritty/a
 2. Edit the Alacritty config file by adding:
 ```yaml
 import:
-  - ./omni.yml
+  - /omni.yml
```

Comment:
```suggestion
  - /omni.yml
```

The actual solution is to use the absolute path to the file!

Can be relative to your home as: `~//.config/alacritty/<fiile>.yml` or the full path: `/home/<user>/.config/alacritty/<file>.yml`